src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1207,6 +1157,9 @@ namespace WaterToAirHeatPumpSimple {
 
         auto &ZoneEqSizing(state.dataSize->ZoneEqSizing);
         auto &simpleWatertoAirHP(state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum));
+        auto &FinalZoneSizing(state.dataSize->FinalSysSizing(state.dataSize->CurSysNum));
+        auto &CompanionCoolingCoil(state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionCoolingCoilNum));
+        auto &CompanionHeatingCoil(state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionHeatingCoilNum));
```

Comment:
This worries me a bit. If this coil type is used in UnitarySystem and the companion coil is not this same type, an array bounds crash will occur. These should be moved to where "if (simpleWatertoAirHP.CompanionCooling/HeatingCoilNum)) protects these references.